core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -379,6 +379,8 @@ public void stopPartitions(Set<StopPartition> stopPartitions,
                         LOGGER.info("Deleting the remote log segments task for partition: {}", tpId);
                         deleteRemoteLogPartition(tpId);
                     }
+                } else {
+                    LOGGER.warn("StopPartition call is not expected for partition: {}", tp);
```

Comment:
tp -> tpId

Also, remove the comments from L368-L370 since we are addressing it in this patch.

Reply:
`tpId` is `TopicIdPartition`, in the warn log, `topicIdByPartitionMap` not contains `tp`, how to get the `tpId`? @kamalcph

Reply:
Then, we can keep it as `tp` itself.